hs628_tools/Test.py

```diff
@@ -0,0 +1,133 @@
+
+import numpy as np
+from sklearn.model_selection import train_test_split
+
+
+
+
+
+
+class data_split:
+    """
+    Create class data_split to predict a perfect train and test data split ratio.
+
+    defines method to accomplish this task.
+
+    returns a split ratio.
+
+    Identifies Missing values as Nan for any missing value, return train and test dataframe with uniform division
+    of NAN between train and test data per the data split ratio.
+
+    """
+    def __init__(self, data):
+        self.data = data
```

Comment:
There is no reason to have self.data = data as you cannot use the self.data datatype further as it is not a default datatype in python. Remember the way we used to work with a Vector/Matrix data type in 608. So usually either you create a work around in your class to adhere to the new datatype for data or just add any other numerical/string values parameters into help.

Reply:
Hi Rohit,

Thanks for your input, I should have had been careful before pushing the code. Thanks for pointing out this to me. You are right, I will go ahead and remove this portion. It was pushed in error. Thanks!

Reply:
@EHWUSF That is correct, since I do not intend to use data later, I decided to omit this part from my package.